sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, default_rqmt, gateway=None, has_memory_resource=True, auto_cl
         :param list[str] ignore_jobs: list of job ids that will be ignored during status updates.
                                       Useful if a job is stuck inside of Slurm and can not be deleted.
                                       Job should be listed as "job_number.task_id" e.g.: ['123.1', '123.2', '125.1']
+        :param job_name_mapping: mapping for job names in sbatch, e.g., path/to/file/JobName.H4sH to JobName.H4sH
```

Comment:
Not sure how to annotate the type here. I'd usually use `typing.Callable`, but it seems like `typing` isn't used in general here.